db/schema.go

```diff
@@ -10,11 +10,44 @@ import (
 type DealStatus string
 
 const (
-	DealStatusPending DealStatus = "PENDING"
-	DealStatusSuccess DealStatus = "SUCCESS"
-	DealStatusFailure DealStatus = "FAILURE"
+	DDM_StorageDealStatusPending            DealStatus = "DDM_PENDING" // Initial state set by DDM (not from Boost)
```

Comment:
Suggestion: If you're ok with importing the storagemarket library, you can just use the constants there instead of redeclaring it here.

```
github.com/filecoin-project/go-fil-markets/storagemarket
```